aggregator/src/aggregation/decoder/witgen/types.rs

```diff
@@ -73,21 +76,21 @@ impl RomTagTableRow {
 /// The symbol emitted by FSE table. This is also the weight in the canonical Huffman code.
 #[derive(Clone, Copy, Debug, EnumIter, PartialEq, Eq, PartialOrd, Ord)]
 pub enum FseSymbol {
-    /// Weight == 0.
+    ///
     S0 = 0,
-    /// Weight == 1.
+    ///
     S1,
-    /// Weight == 2.
+    ///
     S2,
-    /// Weight == 3.
+    ///
     S3,
-    /// Weight == 4.
+    ///
     S4,
-    /// Weight == 5.
+    ///
     S5,
-    /// Weight == 6.
+    ///
     S6,
-    /// Weight == 7.
+    ///
     S7,
 }
```

Comment:
I think we should completely remove this `FseSymbol` type, since the symbol will be a `u64` or `usize` (because the symbols from the FSE tables in sequences section are not bounded by `S7`, they can go much higher)

Reply:
Removed.